TFT/src/User/API/parseACK.c

```diff
@@ -662,21 +668,12 @@ void parseACK(void)
         }
         hasFilamentData = true;
       }
-      else if (infoMachineSettings.onboardSD == ENABLED &&
-               ack_seen(infoMachineSettings.firmwareType != FW_REPRAPFW ? "File opened:" : "job.file.fileName"))
+      else if (infoMachineSettings.onboardSD == ENABLED && ack_seen("File opened:"))
       {
         char * fileEndString;
-        if (infoMachineSettings.firmwareType != FW_REPRAPFW)
-        {
-          // Marlin
-          fileEndString = " Size:";  // File opened: 1A29A~1.GCO Size: 6974
-        }
-        else
-        {
-          // RRF
-          ack_seen("result\":\"0:/gcodes/");  // {"key":"job.file.fileName","flags": "","result":"0:/gcodes/pig-4H.gcode"}
```

Comment:
It looks like this was removed without a corresponding handler to set the job file name?

Can we create a new parser to handle this message and set the print job name?

Reply:
> It looks like this was removed without a corresponding handler to set the job file name?
> 
> Can we create a new parser to handle this message and set the print job name?

My mistake :P  Looks like I'll have to put it in `resq`.